### PR TITLE
Added step for parsing response body

### PR DESCRIPTION
**Pull request by KirillBogdanets**

### CODE REVIEW

File: src/apiActionSteps.ts [hunk 1]
```diff
@@ -277,19 +274,35 @@ When(
...
+ * MANDATORY STEP THAT SHOULD BE USED AFTER SENDING REQUEST
+ * Parsing body in needed way
+ *
+ * @example
+ * I parse "$response" body as "json"
+ *
+ * @param response key of the remembered response
+ * @param type response body parsing type (arrayBuffer|formData|blob|json|text)
+ */
+When('I parse {string} body as {bodyParsingType}', async function (response: string, type: string) {
```
Comment (AlexGalichenko) [inline src/apiActionSteps.ts:303]: I would keep type as string instead of custom as it supports quotes and double quotes

**Review by AlexGalichenko — CHANGES_REQUESTED** [review]

File: src/apiActionSteps.ts (hunk 1 above)
Comment (AlexGalichenko) [reply src/apiActionSteps.ts:303]: And we need to update api-service dependency, currently it still targeting to 0.0.3

Comment (KirillBogdanets) [reply src/apiActionSteps.ts:303]: updated version of the service.

Don't think that use `string` as a parameter type for that variable is the best way to control it. From my point of view if we use custom parameter type than we avoid possible issue of typo or wrong body parser usage that we are not supporting.

**Review by nomid0612 — APPROVED** [review]

File: src/apiActionSteps.ts (hunk 1 above)
Comment (KirillBogdanets) [reply src/apiActionSteps.ts:303]: resolved

**Pull request merged**